src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  if (!sourceString.trim()) {
+    return {};
+  }
+
+  return sourceString
+    .split(';')
+    .map((keyValue) => keyValue.split(':').map((str) => str.trim()))
+    .filter(([key, value]) => key && value)
+    .reduce((accumulator, [key, value]) => {
+      accumulator[key] = value;
```

Comment:
Assigning the value to the accumulator object's key is done correctly. This will correctly build the object with CSS property names as keys and their corresponding values.